app/views/propertyrepresentation/revokeAgentProperties.scala.html

```diff
@@ -37,11 +37,11 @@
 )(implicit request: AuthenticatedRequest[_], messages: Messages, config: ApplicationConfig)
 
 @title = @{
-    Messages("propertyRepresentation.removeAgentProperties.title")
+    messages("propertyRepresentation.removeAgentProperties.heading", model.agentGroup.companyName)
```

Comment:
so is it a title or a heading?
Heading is the `h1` tag, big text at top of the page. `title` is the text displayed in the browser tab. here we're assigning a heading to `@title` but sometimes `@title` is mis-named and is actually a heading... if it's a heading, can we change `@title` to `@heading` ?

Reply:
They are the same thing in this case, so I used the similar existing .heading message for the title. You're right that it's ambiguous, if you feel it's better off as two duplicate messages for clarity, I can change it.

Reply:
'title' and 'heading' are supposed to be the same text on all our pages (except that 'title' has the 'VOA ...' suffix). I don't think we want/need two duplicate messages. I prefer 'title' (in the message file) as that is the non-technical term for the page - i.e. page title - the title then gets used as the browser tab title and the html page heading. But we don't seem to have an agreed convention for this.

Reply:
(I'm happy with the way it is in this PR)